src/index.js

```diff
@@ -19,6 +19,18 @@ if (
 ) {
   window.localStorage.clear()
   window.localStorage.setItem(PACKAGE_VERSION_KEY, PACKAGE_VERSION)
+
+  // Attempt to clean up indexedDB storage as well
+  if (
+    window.indexedDB &&
+    window.indexedDB.databases &&
+    window.indexedDB.deleteDatabase
```

Comment:
Note that firefox is [still waiting to get support for `indexedDB.databases()`](https://bugzilla.mozilla.org/show_bug.cgi?id=934640)